CHANGELOG.rst

```diff
@@ -1,5 +1,41 @@
 .. _changelog:
 
+0.18.0
+------
+
+This release brings improvements and bug fixes to Renku UI.
+
+Features
+~~~~~~~~
+
+* **UI** reduce unnecessary 'project locked' notifications (`#1982 <https://github.com/SwissDataScienceCenter/renku-ui/issues/1982>`_, `#2025 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2025>`_)
+* **UI** expose project metadata as JSON-LD (`#1867 <https://github.com/SwissDataScienceCenter/renku-ui/issues/1867>`_, `#2022 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2022>`_)
+
+Bug fixes
+~~~~~~~~~~
+[fill out or delete before release]
+
+Misc
+~~~~
+[fill out or delete before release]
+
+
+Individual components
+~~~~~~~~~~~~~~~~~~~~~~
+
+- `renku-ui 2.9.0 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/2.9.0>`_
+
+Upgrading from 0.17.x
+~~~~~~~~~~~~~~~~~~~~~~
+
+BREAKING CHANGES!
```

Comment:
We need to mention that after upgrading, this makes any currently running anonymous sessions unavailable. This occurs when going from 0.16.x to 0.17.x as things stand right now.

This happens because the ui-server is now setting the anonymous user id and the old user ids set by the gateway are ignored. And any existing anonymous user id cookies are ignored.

I tested this by deploying 0.16.0 (same as on renkulab.io), starting anonymous sessions and upgrading to the latest on renku's master. After I did this I could not see the anonymous sessions anymore at all in the browser.

Reply:
...This was supposed to be request changes. I misclicked.

Reply:
what is the default time limit we have on anonymous sessions? Maybe we can just disable them for that amount of time before the upgrade.

Reply:
So actually, that's also a problem on the 0.17.2 we are about to roll out. Is that a showstopper?

Reply:
Yes, this will also occur on 0.17.2, but we decided that this is not a problem as long as we do not roll the upgrade out during a workshop of live tutorial session.

We should probably check that there are only at most a few anonymous sessions when make the upgrade.